### ISSUE TITLE
Move version history into separate changelog file

### ISSUE THREAD

**Issue by dreese** [dreese/FastSocket#30]
Having a separate `CHANGELOG` file leads to a CocoaPods page with two tabs at the top. See:
https://cocoapods.org/pods/OHHTTPStubs/

Also, as the change log gets longer, it takes up a lot more of the space that should be devoted to how to use the current version of the library.

https://guides.cocoapods.org/making/quality-indexes

**Comment by dreese** [comment]
Current "score" is below. I'd like to see if it improves after making this change.
```
{
   "github": {
      "subscribers": 17,
      "stargazers": 132,
      "forks": 32,
      "contributors": 2,
      "open_issues": 5,
      "open_pull_requests": 3,
      "created_at": "2014-06-09 03:25:41 UTC",
      "updated_at": "2017-10-05 17:33:55 UTC",
      "language": "Objective-C",
      "closed_issues": 14,
      "closed_pull_requests": 2
   },
   "cocoadocs": {
      "install_size": 48,
      "total_files": 4,
      "total_comments": 313,
      "total_lines_of_code": 504,
      "doc_percent": null,
      "readme_complexity": 71,
      "initial_commit_date": "2017-10-06 01:58:45 UTC",
      "rendered_readme_url": "http://cocoadocs.org/docsets/FastSocket/1.4/README.html",
      "created_at": "2015-03-08 06:50:31 UTC",
      "updated_at": "2017-10-06 08:04:00 UTC",
      "license_short_name": "MIT",
      "license_canonical_url": "http://opensource.org/licenses/MIT",
      "total_test_expectations": 190,
      "dominant_language": "Objective C",
      "quality_estimate": 69,
      "builds_independently": false,
      "is_vendored_framework": null,
      "rendered_changelog_url": null,
      "rendered_summary": "<p>A fast, synchronous Objective-C wrapper around BSD sockets for iOS and OS X.\nSend and receive raw bytes over a socket as fast as possible. Includes methods\nfor transferring files while optionally computing a checksum for verification.<\/p>\n\n<p>Use this class if fast network communication is what you need. If you want to\ndo something else while your network operations finish, then an asynchronous\nAPI might be better.<\/p>",
      "spm_support": false
   },
   "stats": {
      "download_total": 33104,
      "download_week": 0,
      "download_month": 521,
      "app_total": 267,
      "app_week": 0,
      "tests_total": 34,
      "tests_week": 0,
      "created_at": "2015-05-25 09:24:33 UTC",
      "updated_at": "2017-11-03 09:12:53 UTC",
      "extension_week": 0,
      "extension_total": 3,
      "watch_week": 0,
      "watch_total": 0,
      "pod_try_week": 0,
      "pod_try_total": 0,
      "is_active": true
   }
}
```

**Comment by dreese** [comment]
Fixed in #32.
